### ISSUE TITLE
[syncfusion_flutter_pdfviewer] Cannot successfully read some PDF from network

### ISSUE THREAD

**Issue by claireliu14** [syncfusion/flutter-widgets#49]
════════════ Exception caught by widgets library ═════════════════
type '_PdfNull' is not a subtype of type '_PdfDictionary' in type cast
The relevant error-causing widget was: 
  SfPdfViewer file:///Users/claireliu/Desktop/meet_taipei_flutter/lib/page/intro/pdf_viewer_page.dart:15:23
═════════════════════════════════════════════════════

```dart
        SfPdfViewer.network(
            "https://osensetech.blob.core.windows.net/meettaipei/IC-07_poster.pdf",
            canShowScrollHead: false,
            canShowScrollStatus: false,
            canShowPaginationDialog: false,
            enableDoubleTapZooming: false,
            onDocumentLoadFailed: (detail) {
              print(detail.error);
            },
          );
```

**Comment by dillibabun** [comment]
Hi @claireliu14 

We have confirmed that the reported issue with "Type cast exception is thrown while accessing a bookmark instance of PDF document" is a defect. The issue fix will be included in our next weekly package release which is expected on 17 November 2020. The status of this issue can be tracked from the following feedback link.
 

https://www.syncfusion.com/feedback/19606/bookmark-parsing-is-not-working-in-particular-pdf-document

 Regards,
Dilli babu.

**Comment by claireliu14** [comment]
Issue fixed on [syncfusion_flutter_pdfviewer 18.3.50-beta](https://pub.dev/packages/syncfusion_flutter_pdfviewer)
